core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.html

```diff
@@ -563,6 +565,9 @@
   }
 
   @media only screen and (max-width: 830px) {
+    .oppia-topics-and-skills-dashboard {
+      z-index: 10000;
```

Comment:
Can you please explain this requirement?

Reply:
Wanted the lower menu(the topic/skills tab) to look like it's above the top-most green navbar and a lower z-index value was not working. 
![image](https://user-images.githubusercontent.com/30312043/89250931-7b634180-d633-11ea-8d26-16a914520cf2.png)